pkg/search/parser_test.go

```diff
@@ -52,6 +60,32 @@ func TestSplitQuery(t *testing.T) {
 	}
 }
 
+func TestSplitCommaSeparateValues(t *testing.T) {
```

Comment:
Not sure it matters, but this is a deviation from strings.Split
```
		{
			commaSeparatedVals: ",,",
			splitValues:        []string{"", "", ""},
		},
```

Reply:
It shouldn't matter because `splitCommaSeparatedvalues` is always called after `parsePair` and it handles strings ending with comma. But I added a fix for that case for consistency.